### ISSUE TITLE
Typified by complete penetrance

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#9902]
**Preferred term label:**
Typified by complete penetrance

**Synonyms**

**Definition (free text, please give PubMed ID)**
Description of conditions in which all individuals with a given genotype exhibit the disease within a lifespan of 80 years. For example, penetrance of Neurofibromatosis type 1 due to NF1 is close to 100%.
comment: Penetrance describes the proportion of genotype positive individuals that develop disease given a lifespan of 80 years.  

**Parent term (use hpo.jax.org/app)**
Inheritance qualifier HP:0034335

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@angharadroberts 
added as HP:0034950
